components/create/CreateNft.vue

```diff
@@ -0,0 +1,389 @@
+<template>
+  <div class="is-centered columns">
```

Comment:
use `columns` but where are the `column`?

Reply:
in the next line `<form class="is-half column" @submit.prevent="createNft">`